ncm-network/src/main/pan/components/network/config-nmstate.pan

```diff
@@ -13,4 +13,4 @@ prefix "/software/components/network";
 prefix '/software/packages';
 'nmstate' = dict();
 prefix "/system/aii/osinstall/ks";
-"NetworkManager-config-server" = dict();
+'packages' = append("NetworkManager-config-server");
```

Comment:
Probably unnecessary, but for extra safety:
```suggestion
'packages' ?= list();
'packages' = append(SELF, 'NetworkManager-config-server');

```

Reply:
that should not be needed at all ever. single argument append does it all.

Reply:
True, but the explicit `SELF` guards against users adding more items to the append.
Anyway, it's perfectly fine and now merged.